### DISCUSSION TITLE
I am a undergraduate student of Peking University, and I cannot apply for the student pack.

### DISCUSSION THREAD

**Discussion by yudshj** [community/community#19398] [GitHub Education] [answered]
What made me confused is that I uploaded an image with a dated school ID. But GitHub still rejected my application for:

```
Your application is more likely to be quickly reviewed and approved if you address:
    Your image should be one of the following:
    Dated school ID
```

Could anyone give me suggestions, please? I have tried more than 20 times. 🤝🤝🤝

**Comment by melvincwng** [comment, marked as answer]
@maghsk, instead of using your school ID, try reapplying for the Github Education Student Pack with one or more of the following:
1. Class schedule
2. Academic transcript
3. Letter stating your affiliation with an academic institution or enrollment verification letter

If all fails, contact [Github Education Support](https://support.github.com/contact/education) over here by raising a ticket. They will investigate and get back to you usually within a few working days.

Hope this clarifies.

If this reply is helpful, please 'Mark as answer' so the others are aware that this issue has been addressed.

Thank you, and have a great day ahead 😄 

Reference: [Github Docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-for-your-schoolwork/apply-for-a-student-developer-pack)

**Reply by yudshj** [reply]
Thank you melvincwng! I successfully applied for a GitHub Student Pack! 🎉🎉🎉

**Reply by melvincwng** [reply]
@maghsk, hi Donald, congratulations in successfully applying for the Github Student Pack!

Pleasure to have helped you 😄

**Comment by yudshj** [comment]
What I have used for the application:
- School ID card
- Official enrollment letter **(dated)**

If anyone is experiencing the same issue, try taking a photo containing both of the above items as the application material.

**Reply by Jaciezyt** [reply]
(offtopic)膜拜北大大佬

**Reply by melvincwng** [reply]
@Jasonzyt, 哈哈，不管怎样，欢迎来到 GitHub 社区 👋🏻

**Comment by Georgefwt** [comment]
Hello, I am also a university student from China and would like to ask if the Chinese language on the enrollment letter will affect my application. I don't have a English version of that.

**Reply by melvincwng** [reply]
@Georgefwt, my suggestion would be to just apply. If there are issues, GitHub Education would let you know.

Of course, the best way is to provide documentation in English.

Hope this clarifies.

**Reply by Georgefwt** [reply]
> @Georgefwt, my suggestion would be to just apply. If there are issues, GitHub Education would let you know.
> 
> Of course, the best way is to provide documentation in English.
> 
> Hope this clarifies.

Can I ask if I need to take a photo of the whole face? It is difficult to put the whole face in one photo and make sure the words on the certificate can be read clearly. Thank you.

**Reply by melvincwng** [reply]
@Georgefwt, based on my knowledge, actually there should be a button somewhere in the application form that allows you to upload your documents directly.

Can you check if it's there?

Else, you might have to check with GitHub Education Support on this. They are the relevant team that would be able to address this issue.
